tests/blossom.lisp

```diff
@@ -224,18 +235,19 @@ NOTE: This macro automatically rescales the pairs in `COORDINATES' to reside at
                        (push time times)
                        (unless matching
                          (error "No matching produced."))
-                       (unless (one-of matching
-                                       ,@(loop :for correct-matching :in solutions
-                                               :collect `',correct-matching))
-                         (unless (perfect-matching? matching ',coordinates)
-                           (error "Found an imperfect matching: ~A" matching))
-                         (when (< (matching-weight matching) ,solution-weight)
-                           (error "Found better perfect matching: ~A has weight ~A<~A"
-                                  matching
-                                  (matching-weight matching)
-                                  ,solution-weight))
-                         (format t "~%Found perfect matching not in solutions: ~A~%"
-                                 matching))
+                       (unless (perfect-matching? matching ',coordinates)
+                         (error "Found an imperfect matching: ~A" matching))
+                       (when (< (matching-weight matching) ,solution-weight)
```

Comment:
Should this just be `!=`? Also should it be an `assert` rather than when-error?

Reply:
I think both of those suggestions are reasonable, not sure why I chose to go with `error` originally